### ISSUE TITLE
Gin Gonic sessions still using old path for two libraries

### ISSUE THREAD

**Issue by smaheshs** [gin-contrib/sessions#62]
Hi,

I see two issues when we try to download and build our project with gin gonic sessions.

a) Sessions is still using "github.com/boj/redistore" whereas this module has moved to "gopkg.in/boj/redistore.v1" due to which it is not sure if some fixes in the new path are there in the old path or not

b) gin-contrib/sessions/redis/redis.go still refers to "github.com/garyburd/redigo/redis" whereas this library as moved to "github.com/gomodule/redigo/redis"

If these two could be updated, it would avoid manual change whenever moving to a new system.

This is my first query in github.com and so I am sorry if this doesn't classify as an issue or it has to be raised in a different way.

**Comment by bjm88** [comment]
Yes this seems like major issue, cannot use the redis session store (I imagine the very popular way to manage sessions) . Currently getting this error

# github.com/gin-contrib/sessions/redis
/Users/ben/go/src/github.com/gin-contrib/sessions/redis/redis.go:52:42: cannot use pool (type *"github.com/garyburd/redigo/redis".Pool) as type *"github.com/gomodule/redigo/redis".Pool in argument to redistore.NewRediStoreWithPool

**Comment by sisyphus1993** [comment]
> Yes this seems like major issue, cannot use the redis session store (I imagine the very popular way to manage sessions) . Currently getting this error
> 
> # github.com/gin-contrib/sessions/redis
> /Users/ben/go/src/github.com/gin-contrib/sessions/redis/redis.go:52:42: cannot use pool (type *"github.com/garyburd/redigo/redis".Pool) as type *"github.com/gomodule/redigo/redis".Pool in argument to redistore.NewRediStoreWithPool

can you tell me how do you solve it ?

**Comment by bjm88** [comment]
The most reason main branches of project have been updated.  Just using go clean/install didn't work for me though.  I had to go into GOPATH / GOROOT dirs and physically remove the pkgs for the related libraries and then do reinstall and issue is gone now.

**Comment by thinkerou** [comment]
ref: #64
